marko.json

```diff
@@ -69,7 +69,7 @@
         "@hidden": "boolean",
         "@aria-text": "string",
         "@aria-label-close": "string",
-        "@dismissible":"boolean",
+        "@dismissible": "boolean",
```

Comment:
This change will happen from prepublish/postpublish processing, so might as well sneak it in.